packages/platform/src/__tests__/platform.test.ts

```diff
@@ -128,10 +128,42 @@ describe('platform', () => {
 
     platform.loadMetadata(ids.meta, {
       M1: 'hey',
-      M2: 'there'
+      M2: 'there',
     })
```

Comment:
trailing comma does not conform to `standard` code style https://standardjs.com we use in the project. Use `yarn lint` to check style before commit.